stdlib.sh

```diff
@@ -104,6 +110,14 @@ EOC
 #
 # ... near the top of the calling script.
 
+declare std_PRETEND
+# Standard usage is:
+#
+std_PRETEND="${PRETEND:-0}"
```

Comment:
As you prefer. I used PRETEND because it is what we used in one of the first versions of stdlib.sh.